rpc/client/mock/abci.go

```diff
@@ -88,7 +88,6 @@ func (a ABCIApp) BroadcastTxAsync(ctx context.Context, tx types.Tx) (*coretypes.
 	return &coretypes.ResultBroadcastTx{
 		Code:      c.Code,
 		Data:      c.Data,
-		Log:       c.Log,
 		Codespace: c.Codespace,
 		Hash:      tx.Hash(),
 	}, nil
```

Comment:
Do we actually remove `Log` from `ResultBroadcastTx` somewhere?

Reply:
I just removed it. Can you verify that this (`ResultBroadcastTx`) refers only to `CheckTx` result